docs/VnVPlan/VnVPlan.tex

```diff
@@ -259,48 +260,555 @@ \subsection{Tests for Functional Requirements}
+  
+  \item{TFR3-A4\\}
+  
+  Control: Functional, Manual, Dynamic
+            
+  Initial State: The user has logged in, and a drop-down menu is clicked and expanded
+            
+  Input: The option \textit{log out} in the drop-down menu is selected and clicked
+            
+  Output: The user should be able to log out of his/her account and prompted saying that you have logged out successfully
+  
+  Test Case Derivation: If the option \textit{log out} is selected and clicked, the user should be able to log out of his/her account and the user should be notified in some way so that he/she knows that the account has been logged out successfully
+  
+  How test will be performed: Manually log in using a mock account using different browsers and then log out for each one of them and see if we can log out and receive notifications about logging out
+  \item{TFR4-A5\\}
+  
+  Control: Functional, Manual, Dynamic
+            
+  Initial State: The user has logged in and in the home page
+            
+  Input:  The tab to view the scheduling information on the home page is selected and clicked
+            
+  Output: The scheduling information (in list view, Kanban view or calendar view, by default it is in list view) is displayed
+  
+  Test Case Derivation: The user should be able to view his/her scheduling information in a reasonable format such as list view, Kanban view or calendar view if the option to view the scheduling information is clicked on the home page
+            
+  How test will be performed: Manually log in using a list of mock accounts (the difference between these accounts would be the customized way of viewing the scheduling information such as by default using list view, using Kanban view, or using calendar view) using different browsers and then click the tab that is responsible for redirecting to the page viewing the scheduling information on the home page and check if we can be redirected to the page listing scheduling information and check if the information is displayed in the view that the account has been set
+  
+  \item{TFR5-A6\\}
+  
+  Control: Functional, Manual, Dynamic
+            
+  Initial State: The user is on the page where he can retrieve his password by answering a list of security questions
+            
+  Input:  Strings to each one of the security questions that match the user data stored in our database
+            
+  Output: The password gets changed successfully and updated in our database, a text or diagram should be displayed to notify the user that the password has been changed 
```

Comment:
I think text should be good, can't think about what kind of  diagram should be displayed.